test/calibration/experiments/test_rabi.py

```diff
@@ -89,6 +92,7 @@ def test_wrong_processor(self):
         rabi.analysis.set_options(data_processor=DataProcessor(fail_key, []))
         rabi.set_run_options(shots=2)
         data = rabi.run(backend)
+        self.assertFail(data)
```

Comment:
This should explicitly assert that the status is error if that is what you are testing for.